arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -3751,6 +3951,38 @@
       },
       "description": "The content hash."
     },
+    "RetryHistory": {
+      "type": "object",
+      "properties": {
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets the start time."
```

Comment:
I think @ravbhatnagar put it well. 

@shimedh - I'll be okay to merge this, but please do create a work item and track it.